examples/sls/.nvmrc

```diff
@@ -1 +1 @@
-16.20.0
```

Comment:
tu as oublié de changer le engine dans le root package.json.

Reply:
Ça explique l'erreur de la CI !

Et je t'en conjure @Sc0ra , configure ton éditeur de code (Visual Studio Code) pour mettre des newlines at the end of the files.

Cdt,
Guigui